docs/API/V1/list.md

```diff
@@ -0,0 +1,118 @@
+(regardless of ownership).
+
+`end` date/time \
+Select only datasets created on or before the specified time. Time should be
+specified in ISO standard format, as `YYYY-MM-DDThh:mm:ss.ffffff[+|-]HH:MM`.
+If the timezone offset is omitted it will be assumed to be UTC (`+00:00`); if
+the time is omitted it will be assumed as midnight (`00:00:00`) on the
+specified date.
+
+`limit` integer \
+"Paginate" the selected datasets by returning at most `limit` datasets. This
+can be used in conjunction with `offset` to progress through the full list in
+smaller chunks either to assist with a paginated display or to limit data
+transfer requirements.
+
+`metadata` list \
+Request the named [metadata](../metadata.md) to be returned along with the
+resource name and ID. This can be a single metadata key path or comma-separated
+list of such strings. The `metadata` query parameter can be repeated. For
+example, the following are all equivalent:
+* `?metadata=dataset.created,server.deletion,user`
+* `?metadata=dataset.created&metadata=dataset.deletion,user`
+* `?metadata=dataset.created&metadata=dataset.deletion&metadata=user`
+
+`offset` integer \
+"Paginate" the selected datasets by skipping the first `offset` datasets that
+would have been selected by the other query terms. This can be used with
+`limit` to progress through the full list in smaller chunks either to assist
+with a paginated display or to limit data transfer requirements.
+
+`owner` string \
+The `username` property of a registered user. Only datasets owned by that user
+will be included in the list.
+
+`start` date/time \
+Select only datasets created on or after the specified time. Time should be
+specified in ISO standard format, as `YYYY-MM-DDThh:mm:ss.ffffff[+|-]HH:MM`.
+If the timezone offset is omitted it will be assumed to be UTC (`+00:00`); if
+the time is omitted it will be assumed as midnight (`00:00:00`) on the
+specified date.
+
+## Request headers
+
+`authorization: bearer` token [_optional_] \
+*Bearer* schema authorization is required to access any non-public dataset.
+E.g., `authorization: bearer <token>`. If omitted, the client is unauthenticated
+and only `public` access datasets will be selected.
+
+## Response headers
+
+`content-type: application/json` \
+The return is a serialized JSON object with information about the selected
+datasets.
+
+## Response status
+
+`401`   **UNAUTHORIZED** \
+The client did not provide an authentication token and there is no public
+dataset with the name `<dataset>`.
+
+`403`   **FORBIDDEN** \
+The named `<dataset>` is not public and the authenticated user lacks
+authorization to read it.
+
+`404`   **NOT FOUND** \
+Either the `<dataset>` or the relative `<path>` within the dataset does not
+exist.
+
+## Response body
+
+This returns an `application/json` response body containing a list of objects
+which describe the datasets selected by the specified query criteria.
```

Comment:
s/This returns/This API returns/

(Or, perhaps, more properly, instead of "API", use "method"? Although, if you make that change here, you should do the same in the analogous places in the other methods, as well....)